frontend/src/libs/components/icon/icon.tsx

```diff
@@ -5,12 +5,25 @@ import { iconNameToIcon } from './libs/maps/icon-name-to-icon.map.js';
 type Properties = {
   name: IconName;
   color?: string;
+  width?: number;
+  height?: number;
 };
 
-const Icon: React.FC<Properties> = ({ name, color = 'currentColor' }) => {
+const Icon: React.FC<Properties> = ({
+  name,
+  color = 'currentColor',
+  width,
+  height,
+}) => {
   const SelectedIcon = iconNameToIcon[name];
 
-  return <SelectedIcon style={{ color }} />;
+  const svgProperties = {
+    color,
+    width: width ? `${width}px` : undefined,
+    height: height ? `${height}px` : undefined,
```

Comment:
We're trying not to set undefined, maybe null if it's allowed or empty single quote